### DISCUSSION TITLE
`placeholder` doesn't seem to do anything for `Select`

### DISCUSSION THREAD

**Discussion by joeyfigaro** [ariakit/ariakit#4309] [Questions]
I can't get values passed to a `Select`'s `placeholder` prop to render for the life of me. I'm currently on `v0.4.5` – it looks like `placeholder` doesn't even exist according to the docs?

The following shows an empty select:
```tsx
import { Select, SelectProvider } from '@ariakit/react' 

const TestSelect = () => {
  return (
    <SelectProvider>
      <Select placeholder="Why?" />
    </SelectProvider>
  )
}
```

![image](https://github.com/user-attachments/assets/12ae84c4-ec52-4227-bb4b-4b331deea3aa)

**Comment by diegohaz** [comment]
Where does this `placeholder` prop come from?

If you describe your specific use case, ideally as close to real-world usage as possible, we can recommend the appropriate API for you.
